network/src/commonTest/kotlin/com/wire/kalium/api/tools/json/model/DomainToUserIdToClientToPreKeyMapJson.kt

```diff
@@ -0,0 +1,58 @@
+package com.wire.kalium.api.tools.json.model
+
+import com.wire.kalium.api.tools.json.ValidJsonProvider
+import com.wire.kalium.network.api.prekey.DomainToUserIdToClientsToPreykeyMap
+import com.wire.kalium.network.api.prekey.PreKey
+
+object DomainToUserIdToClientToPreKeyMapJson {
+
+    private const val DOMAIN_1 = "domain1.example.com"
+    private const val DOMAIN_2 = "domain2.example.com"
+
+    private const val USER_1 = "user10d0-000b-9c1a-000d-a4130002c221"
+    private const val USER_1_client = "60f85e4b15ad3786"
+    private val USER_1_client_PREYKEY = PreKey(key = "pQABAQECoQBYIOjl7hw0D8YRNq...", id = 1)
+
+    private const val USER_2 = "user200d0-000b-9c1a-000d-a4130002c221"
+    private const val USER_2_client = "32233lj33j3dfh7u"
+    private val USER_2_client_PREYKEY = PreKey(key = "pQABAQECoQBYIOjl7hw0D8YRNq...", id = 1)
```

Comment:
Casing.
Also, just to make sure our tests won't fail to detect a mapping problem in the future, could you make the `USER_2_client_PREYKEY` different from `USER_1_client_PREYKEY`?